commons-test/src/main/java/org/infinispan/commons/test/categories/Smoke.java

```diff
@@ -0,0 +1,7 @@
+package org.infinispan.commons.test.categories;
+
+/**
+ * {@link org.junit.experimental.categories.Category} tag for smoke tests.
+ */
+public class Smoke {
```

Comment:
Is there any advantage to using class/final class/interface as the category class? JUnit wiki uses interface in the example code.

Reply:
Both a class and interface work. This is just a marker that the test class annotated with this type should be picked by the test suite runner. I don't see any difference.

Reply:
Thinking about it a little more. We can build a hierarchy of categories with both interfaces and non-final classes.